apps/pie-microsite/src/tokensMetadata.json

```diff
@@ -426,232 +577,383 @@
+          "globalToken": "$color-orange-55"
         },
         "content-default": {
-          "category": "static"
+          "category": "static",
+          "description": "Default text and icon colour",
+          "globalToken": "$color-white"
         },
         "content-subdued": {
-          "category": "static"
+          "category": "static",
+          "description": "Subdued text and icon colour used for secondary text and input labels",
+          "globalToken": "$color-charcoal-30"
         },
         "content-interactive-brand": {
-          "category": "interactive"
+          "category": "interactive",
+          "description": "Brand colour for interactive text and icons",
+          "globalToken": "$color-orange-30"
         },
         "content-interactive-light": {
-          "category": "interactive"
+          "category": "interactive",
+          "description": "Persistent colour for interactive text and icons on dark backgrounds. Uses the same global token in all themes",
+          "globalToken": "$color-white"
         },
         "content-interactive-primary": {
-          "category": "interactive"
+          "category": "interactive",
+          "description": "Colour for the text and icons of primary interactive elements",
+          "globalToken": "$color-charcoal-80"
         },
         "content-interactive-secondary": {
-          "category": "interactive"
+          "category": "interactive",
+          "description": "Colour for the text and icons of secondary interactive elements",
+          "globalToken": "$color-white"
         },
         "content-interactive-tertiary": {
-          "category": "interactive"
+          "category": "interactive",
+          "description": "Colour for the text and icons of tertiary interactive elements",
+          "globalToken": "$color-white"
         },
         "content-interactive-inverse": {
-          "category": "interactive"
+          "category": "interactive",
+          "description": "Colour for interactive text and icons on inverse backgrounds",
+          "globalToken": "$color-white"
         },
         "content-interactive-dark": {
-          "category": "interactive"
+          "category": "interactive",
+          "description": "Persistent colour for interactive text and icons on light backgrounds. Uses the same global token in all themes",
+          "globalToken": "$color-charcoal-80"
         },
         "content-interactive-error": {
-          "category": "interactive"
+          "category": "interactive",
+          "description": "Colour for the text and icons of error-specific interactive elements",
+          "globalToken": "$color-red-30"
         },
         "content-interactive-subdued": {
-          "category": "interactive"
+          "category": "interactive",
+          "description": "Subdued text and icon colour used for text and icons",
+          "globalToken": "$color-charoal-40"
```

Comment:
```suggestion
          "globalToken": "$color-charcoal-40"
```

Reply:
_sigh_ this was copied straight out of figma